README.md

```diff
@@ -46,11 +46,12 @@ So far, we have reported 5 new bugs/vulnerabilities found by automatically gener
 by this framework:
 | Project |    LLM    | Prompt template |
 | ------- | --------- | --------------- |
-| [`cJSON` ](https://github.com/DaveGamble/cJSON/issues/800) | Vertex AI | [default](prompts/template_xml) |
-| [`libplist`](https://github.com/libimobiledevice/libplist/issues/244) | Vertex AI | [default](prompts/template_xml) |
-| [`hunspell`](https://github.com/hunspell/hunspell/issues/996) | Vertex AI | [default](prompts/template_xml) |
+| [OOB read in `cJSON` ](https://github.com/DaveGamble/cJSON/issues/800) | Vertex AI | [default](prompts/template_xml) |
+| [OOB read in `libplist`](https://github.com/libimobiledevice/libplist/issues/244) | Vertex AI | [default](prompts/template_xml) |
+| [OOB read in `hunspell`](https://github.com/hunspell/hunspell/issues/996) | Vertex AI | [default](prompts/template_xml) |
 | [Undisclosed stack buffer underflow](https://bugs.chromium.org/p/oss-fuzz/issues/detail?id=67483) | Vertex AI | [default](prompts/template_xml) |
 | [Undisclosed heap buffer overflow](https://bugs.chromium.org/p/oss-fuzz/issues/detail?id=67497) | Vertex AI | [default](prompts/template_xml) |
+| [Undisclosed use of unitialised memory](https://bugs.chromium.org/p/oss-fuzz/issues/detail?id=67516) | Vertex AI | [default](prompts/template_xml) |
```

Comment:
s/unitialised/uninitialised/

Reply:
nit: Would this format be more consistent and easier to read?
`cJSON`: OOB read
`libplist`: OOB read
`hunspell`: OOB read

Undisclosed: stack buffer underflow
Undisclosed: heap buffer overflow
Undisclosed: use of unitialised memory

We can also add a new column for reason.

Reply:
Sure! Can you create a PR for this?

Reply:
Done: https://github.com/google/oss-fuzz-gen/pull/175